src/Footer.tsx

```diff
@@ -332,6 +332,7 @@ export const Footer = memo(
                                             )}
                                             target="_blank"
                                             href={`https://${domain}`}
+                                            title={`${domain} - ouvre une nouvelle fenêtre`}
```

Comment:
"ouvre une nouvelle fenêtre" should be localized as well.

Reply:
hi, It's fixed.

Reply:
```suggestion
                                            title={`${domain} - ${t("open new window")}`}
```